doc/03-Configuration.md

```diff
@@ -1,20 +1,16 @@
 # Configuration
 
-1. [Backend](#backend)
-2. [Mail](#mail)
-3. [Permissions](#permissions)
+Icinga Reporting is configured via the web interface. Below you will find an overview of the necessary settings.
 
 ## Backend
 
-If not already done during the installation of Icinga Reporting, setup the reporting database backend now.
+Configuring the Icinga Reporting database backend now.
```

Comment:
~~Configuring the Icinga Reporting~~
That should not be required to repeat here.

Instead, tell what's stored inside the database if you want a description for this section.